File: nativelogger/src/main/java/cn/jesse/nativelogger/util/DateUtil.java
package cn.jesse.nativelogger.util;

import java.text.SimpleDateFormat;

/**
 * Created by jesse on 9/6/16.
 */
public class DateUtil {
    private static final String TEMPLATE_DATE = "yyyy-MM-dd";

    private DateUtil() {
        //unused
    }

    public static String getCurrentDate() {
        SimpleDateFormat sdf = new SimpleDateFormat(TEMPLATE_DATE);
        return sdf.format(System.currentTimeMillis());
    }
}

File: nativelogger/src/main/java/cn/jesse/nativelogger/util/DateUtil.kt
package cn.jesse.nativelogger.util

import java.text.SimpleDateFormat

/**
 * 日期工具类
 *
 * @author Jesse
 */
object DateUtil {
    private val TEMPLATE_DATE = "yyyy-MM-dd"

    /**
     * 以yyyy-MM-dd格式获取当前时间
     */
    fun getCurrentDate(): String {
        val sdf = SimpleDateFormat(TEMPLATE_DATE)
        return sdf.format(System.currentTimeMillis())
    }
}
